app/adapters/application.js

```diff
@@ -2,16 +2,33 @@ import Ember from 'ember';
 import DS from 'ember-data';
 import ENV from '../config/environment';
 
+// matches query params or trailing slash
+const TRAILING_SLASH_PATTERN = /((^.*?[^\/])\?(.*?)$|^[^\?]+?[^\/]$)/;
+
 export default DS.RESTAdapter.extend(DS.BuildURLMixin, {
+    session: Ember.inject.service(),
     namespace: 'api/v2',
     host: ENV.apiBaseUrl,
     headers: {
         Accept: 'application/json'
     },
     ajax(url, method, hash) {
+        // trailing slash required for POST requests
+        let urlRegex = new RegExp(TRAILING_SLASH_PATTERN);
+        if (url.match(urlRegex)) {
```

Comment:
Wouldn't that also add a `/` to the end if there are query parameters?

![screen shot 2016-09-28 at 1 34 15 pm](https://cloud.githubusercontent.com/assets/7131985/18925059/5a6911ce-8580-11e6-9f74-f8018b2c89c3.png)